gbpservice/nfp/core/event.py

```diff
@@ -283,7 +285,7 @@ def _log_meta(self, event_id, event_handler=None):
         else:
             return "(event_id - %s) - (event_handler - None)" % (event_id)
 
-    def register(self, event_id, event_handler):
+    def register(self, event_id, event_handler, module='', priority=0):
```

Comment:
it is module priority, dont see a case where we need event level priority.
Priority will help in deciding which module should handle an overlapping event which is not specifically targeted.